lib/omnibus/packagers/ips.rb

```diff
@@ -81,7 +81,16 @@ def publisher_prefix(val = NULL)
     # @see Base#package_name
     #
     def package_name
-      "#{safe_base_package_name}.p5p"
+      "#{omnibus_package_name}.p5p"
+    end
+
+    #
+    # We want to have a consistent omnibus package name, but we are required
+    # to have the fmri version for the IPS repo creation.
+    #
+    def omnibus_package_name
```

Comment:
What is the purpose of introducing an extra `omnibus_package_name` method here? We should also include an iteration in the package name like we do for all other packages.